index.ts

```diff
@@ -503,7 +557,7 @@ export class EthereumValue {
 export enum ValueKind {
   STRING = 0,
   INT = 1,
-  FLOAT = 2,
```

Comment:
Another breaking change, although this one is more behind the scenes, so I'd be ok with it. We did say we'd replace `Float` with `BigDecimal`.